fleet-packages-quickstart/README.md

```diff
@@ -0,0 +1,246 @@
+```
+gcloud config set project $PROJECT_ID
+```
+
+If you want to use your current project, you can set the environment variable with:
+
+```
+PROJECT_ID=$(gcloud config get-value project)
+```
+
+## Install Config Sync across the fleet
+
+Fleet Packages is built on top of Config Sync, so we need to install Config Sync on all clusters.
+
+Create the apply-spec:
+
+```
+cat << EOF > apply-spec.yaml
+applySpecVersion: 1
+spec:
+  configSync:
+    enabled: true
+EOF
+```
+
+Enable the config-management feature on the fleet and set it up to install Config Sync on all new clusters:
+
+```
+gcloud beta container fleet config-management enable --fleet-default-member-config=apply-spec.yaml --project=$PROJECT_ID
+```
+
+
+## Set up a fleet
+
+We need a few GKE clusters registered to a fleet. We only create two clusters here to keep things simple.
+
+
+Create a cluster in us-west:
+```
+gcloud container clusters create fp-quickstart-cluster-west --project=$PROJECT_ID --zone=us-west2-a --workload-pool=$PROJECT_ID.svc.id.goog --enable-fleet --async
+```
+
+Create a cluster in us-east:
+```
+gcloud container clusters create fp-quickstart-cluster-east --project=$PROJECT_ID --zone=us-east1-c --workload-pool=$PROJECT_ID.svc.id.goog --enable-fleet --async
+```
+
+It will take a few minutes for the clusters to be ready. You can continue with the next steps and there is a later step to verify that they are ready.
+
+
+## Configure Cloud Build Repositories
+
+We have a sample package at https://github.com/GoogleCloudPlatform/anthos-config-management-samples/tree/main/fleet-packages-quickstart/config. This is a simple example that just sets up nginx. Fork this repo into your own GitHub account.
+
+Fleet Packages leverages [Cloud Build Repositories] (2nd gen) for connecting to git. Follow these steps to set it up:
+
+1. Enable the Cloud Build and Secret Manager APIs:
+```
+gcloud services enable cloudbuild.googleapis.com --project=$PROJECT_ID
+gcloud services enable secretmanager.googleapis.com --project=$PROJECT_ID
+```
+
+2. Verify the IAM permissions as described in https://cloud.google.com/build/docs/automating-builds/github/connect-repo-github?generation=2nd-gen#iam_perms.
+
+3. Create a connection to your GitHub host using the steps described in https://cloud.google.com/build/docs/automating-builds/github/connect-repo-github?generation=2nd-gen#connecting_a_github_host. You can use either the console or gcloud. You must use the region `us-central1` and we suggest using the name `fp-quickstart` for the connection (the rest of the guide will use this name).
+
+4. Connect to the forked GitHub repository using the steps described in https://cloud.google.com/build/docs/automating-builds/github/connect-repo-github?generation=2nd-gen#connecting_a_github_repository_2. You can use either the console or gcloud. Again, you must use the region `us-central1`, connection name `fp-quickstart`, and we suggest using the name `anthos-config-management-samples` for the repository.
```

Comment:
Do users have to fork, or can they skip this step to just use the public sample repo for simplicity?

Reply:
They need to fork unfortunately. Cloud Build Repositories installs a GitHub app on the org in which the repo exists.

Reply:
I see. Would you add some clarification that forking it needed? Users might try to skip this step